src/js/components/Header/stories/Sticky.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  Anchor,
+  Avatar,
+  Card,
+  Grid,
+  Header,
+  Nav,
+  Page,
+  PageContent,
+  Paragraph,
+} from 'grommet';
+
+export const Sticky = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Page kind="narrow">
+    <PageContent background="light-3">
+      <Header sticky="scrollup" background="light-4" pad="small">
+        <Avatar background="brand">SY</Avatar>
+        <Nav direction="row">
+          <Anchor label="Home" href="#" />
+          <Anchor label="Profile" href="#" />
+        </Nav>
+      </Header>
+      <Paragraph>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer commodo
+        gravida tincidunt. Nunc fringilla blandit tortor, id accumsan nisi
+        dictum quis. Aenean porttitor at mi id semper. Donec mattis bibendum
+        leo, interdum ullamcorper lectus ultrices vel. Fusce nec enim faucibus
+        nunc porta egestas. Fusce dapibus lobortis tincidunt.
+      </Paragraph>
```

Comment:
If we are going to have a lot of text, would it make sense for us to use it? Just an idea.

```suggestion
      <Paragraph>
        To maximize screen real-estate, the Header on this page scrolls out of view as the user moves down the page. However if the user scrolls upwards, the Header is revealed and fixed atop the window. On long pages this behavior allows easy access to the Header's content, such as navigation or menus, while preventing the Header from obscuring content on mobile devices or in smaller windows.
      </Paragraph>
```